src/Analytics/Plausible.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Utopia PHP Framework
+ *
+ * @package Analytics
+ * @subpackage Tests
+ *
+ * @link https://github.com/utopia-php/framework
+ * @version 1.0 RC1
+ * @license The MIT License (MIT) <http://www.opensource.org/licenses/mit-license.php>
+ */
+
+namespace Utopia\Analytics;
+
+class  PlausibleAdapter extends Analytics
+{
+  private const URL='https://plausible.io/api/event';
+
+
+  public function __construct(string $clientIP){
+    $clientIP=$_SERVER['REMOTE_ADDR'];
+    $this->headers=array(' X_FORWARDED_FOR: '  .$clientIP , ' Content-Type: application/json ');
```

Comment:
For better readability 👍🏻 

```suggestion
    $this->headers = [
        'X_FORWARDED_FOR: '  .$clientIP, 
        'Content-Type: application/json'
    ];
```